i-slide.js

```diff
@@ -122,7 +249,7 @@ class ISlide extends HTMLElement {
     });
 
     if (this.type === 'application/pdf') {
-      this.loadPDFScripts();
+      this.#loadPDFScripts();
```

Comment:
no need to change this in this PR, but really this should move out of `#fetch` into the #type setter  (that way changing the type no longer has a dependency on fetch)

Reply:
Good point. I moved it while I was at it.